src/github.rs

```diff
@@ -865,6 +865,7 @@ impl Repository {
     ) -> String {
         let filters = filters
             .iter()
+            .filter(|(key, val)| !(*key == "state" && *val == "all"))
```

Comment:
```suggestion
            .filter(|&(ref key, ref val)| !(key == "state" && val == "all"))
```

Reply:
@bjorn3 what's the difference in your suggestion? (just curious)

Reply:
My suggestion should actually have been
```suggestion
            .filter(|&(key, val)| !(key == "state" && val == "all"))
```
This removed the explicit dereferences.

Reply:
@bjorn3 thanks for your suggestions, the first one I guess is debatable which one is better.

The second one doesn't work ...

```
error[E0277]: can't compare `&str` with `str`
   --> src/github.rs:868:41
    |
868 |             .filter(|&(key, val)| !(key == "state" && val == "all"))
    |                                         ^^ no implementation for `&str == str`
    |
    = help: the trait `PartialEq<str>` is not implemented for `&str`
    = note: required because of the requirements on the impl of `PartialEq<&str>` for `&&str`
```